README.md

```diff
@@ -1 +1,180 @@
-# awesomebooks-es6
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>AWESOME BOOKS</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome books: Use of Javascript classes] <a name="about-project"></a>
+
+This project is designed to help us understand JavaScript classes and how to utilize it. Also about localStorage and usage of DOM.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML-5
+- CSS3
+- Linters
+- Javascript
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.testim.io/blog/what-is-a-linter-heres-a-definition-and-quick-start-guide/">Linters</a></li>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Key features of the application.
+
+- **[responsiveness]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- An IDE (Intergrated Development Environment ) such as vsode or atom
+- A version control system such as git
+- Node
+
+### Setup
+
+1.  Clone the Repo with:
+
+```
+git clone https://github.com/H-Ramalan/Awesome-Books
+```
+
+2. switch to the cloned directory
+
+```
+cd Awesome-Books
+```
+
+### Installation
+
+1. Install the project with it's depenedcies
+
+```
+npm install
+```
+
+Then open the live server or open the `index.html` file with your browser
+
+### Usage
+
+- this is the capstone project I'll use to publicize the Show.
+
+### Run tests
+
+To run tests, run the following command:
+
+- To run tests run :
+- npx hint .
+- npx stylelint "\*_/_.{css,scss}"
+- npx eslint .
+
+<!-- Deployment -->
+
+### Deployment
+
+- Project is deployed using [GitHub Pages](https://pages.github.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Hamza Ramalan**
+
+- GitHub: (https://github.com/H-Ramalan)
+- Twitter: (https://twitter.com/@hamza_ramalan)
+- LinkedIn: (https://www.linkedin.com/in/hamza-r-3b3b6a/)
```

Comment:
![image](https://github.com/H-Ramalan/awesomebooks-es6/assets/93039359/4a5c4caa-b8ed-48e5-8b9f-570af81613a6)

- [ ] It would be nice if you fix your LinkedIn link because right now it is broken 😨 Kindly fix this link to make your readme descriptive and professional 👍